pkg/kubelet/apis/config/v1beta1/defaults.go

```diff
@@ -149,6 +149,9 @@ func SetDefaults_KubeletConfiguration(obj *kubeletconfigv1beta1.KubeletConfigura
 		// Keep the same as default NodeStatusUpdateFrequency
 		obj.CPUManagerReconcilePeriod = metav1.Duration{Duration: 10 * time.Second}
 	}
+	if obj.TopologyManagerPolicy == "" {
+		obj.TopologyManagerPolicy = "Preferred"
```

Comment:
Is it OK that "Preferred" is capitalized here, but is lower-case in the TopologyManager itself? Does it get normalized to lower-case somewhere?

Reply:
Doesn't seem to make a difference, but changed anyway for consistency.